tutorials/computer-security/tox/tutorial.yml

```
@@ -0,0 +1,19 @@
+id: 027bc897-8c98-4265-b85b-e78b7ab607f3
+project_id: f77762d1-831a-4a95-8808-1143102fc6c3
+professor_id: 81a82a0f-3003-4459-9d6a-67f860efb20e
+level: beginner
+category: communication
+tags:
+  - privacy
+  - protocols
+  - personal-security
+# Proofreading metadata
+
+original_language: fr
+proofreading:
+  - language: fr
+    last_contribution_date: 2025-07-09
+    urgency: 1
+    contributors_names:
```

```suggestion
    contributor_names:
```
Careful, there's an extra "s" here